packages/utilities/gel-constants/breakpoints.js

```diff
@@ -1,36 +1 @@
-/* 
-   Screen sizes for GEL Typography
-   These namings are based on the GEL description. They are also known as group B and group D
-   Link to relevant GEL docs: http://www.bbc.co.uk/gel/guidelines/typography#type-sizes
-*/
-export const GEL_GROUP_B_MIN_WIDTH = 20; // 320px
-export const GEL_GROUP_B_MAX_WIDTH = 37.4375; // 599px
-export const GEL_GROUP_CD_MIN_WIDTH = 37.5; // 600px
-
-/*
-    The following are breakpoints from GEL Grid
-    Link to relevant GEL docs: https://www.bbc.co.uk/gel/guidelines/grid#grid-sizes
-    The only exception is that we have split out group 1 into 0 and 1
-*/
-export const GEL_GROUP_0_SCREEN_WIDTH_MIN = `0em`; // 0px
-export const GEL_GROUP_0_SCREEN_WIDTH_MAX = `14.9375em`; // 239px
-
-export const GEL_GROUP_1_SCREEN_WIDTH_MIN = `15em`; // 240px
-export const GEL_GROUP_1_SCREEN_WIDTH_MAX = `24.9375em`; // 399px
-
-export const GEL_GROUP_2_SCREEN_WIDTH_MIN = `25em`; // 400px
-export const GEL_GROUP_2_SCREEN_WIDTH_MAX = `37.4375em`; // 599px
-
-export const GEL_GROUP_3_SCREEN_WIDTH_MIN = `37.5em`; // 600px
-export const GEL_GROUP_3_SCREEN_WIDTH_MAX = `62.9375em`; // 1007px
-
-export const GEL_GROUP_4_SCREEN_WIDTH_MIN = `63em`; // 1008px
-export const GEL_GROUP_4_SCREEN_WIDTH_MAX = `80em`; // 1279px
-
-export const GEL_GROUP_5_SCREEN_WIDTH_MIN = `80em`; // 1280px
-
-export const MEDIA_QUERY_TYPOGRAPHY = {
-  SMART_PHONE_ONLY: `@media (min-width: ${GEL_GROUP_B_MIN_WIDTH}em) and (max-width: ${GEL_GROUP_B_MAX_WIDTH}em)`,
-  SMART_PHONE_AND_LARGER: `@media (min-width: ${GEL_GROUP_B_MIN_WIDTH}em)`,
-  LAPTOP_AND_LARGER: `@media (min-width: ${GEL_GROUP_CD_MIN_WIDTH}em)`,
-};
+module.exports = require('./dist/breakpoints'); // eslint-disable-line import/no-unresolved
```

Comment:
Does material UI just dyamically build a new version of the packages? sorry not sure which part your referring to

Reply:
Im not sure how we could do that when it comes to using lerna. You can use a lerna command to link all packages togeather, but that might not work if we generate the files somewhere else. And if we do it in place it might get a bit crazy with .gitignores to make sure we dont put them in git. If you can think of a nice way to do it though im all ears :)

Reply:
Bare in mind this is only for these constants libraries where we have chosen to address them using `@bbc/psammead-styles/colours`, pretty much every normal component wont need to do this

Reply:
I guess we'd define a `build` command in the root-level `package.json` which goes through all the components and transpiles them.

```
packages/
  my-package/
    src/
      index.jsx
    dist/ # generated by top-level 'build' command. git-ignored
    package.json
```

And the component-level package.json would have `main: "dist/index.js"` and `files: ["dist"]` properties. That way it should be possible for consumers to write `import foo from "@bbc/my-package"` as well as `import bar from "@bbc/my-package/fonts"`.

Only downsides I can think of are 1) symlinking (need to run the `build` command when debugging locally) and 2) unable to pull in untranspiled versions for modern browsers (unless we expand the `files` property to `files: ["dist", "src"]`).

Upsides are that we don't have to change the way we write components - it's still ES6, it's just down a level (in `src/`). And we can still use Lerna to manage deployments (would be `npm run build && lerna publish --independent` from the root).

Reply:
Running it locally I cant get `"files": ["dist"]` to work, is it only when publishing? Mine only works when that `module.exports = require('./dist/colours');` file is at the base dir

Reply:
Looking at https://docs.npmjs.com/files/package.json#files it looks as if this doesnt dictate what files it points at, just what are packaged up

Reply:
What I mean is if `main: "dist/index.js`, and we then `npm link` the module,
then in our application we `npm link @bbc/psammead-foo`,
when we `import foo from "@bbc/psammead-foo"` then it will try to pull in `dist/index.js` - but `dist/index.js` doesn't exist until you run the `build` step at the root of Psammead.

Reply:
Yeah thats fine, the reason for these ES5 files is that were not using an index.js for these packages. If we did ```import { colours } from '@bbc/psammead-styles'``` we could, but as were going with `import { foo } from '@bbc/psammead-styles/colours'``` we cant use an index.js and thus need this pass through

Reply:
@ChrisBAshton @dr3 I'd suggest merging this PR as it is, since it works well and provides the API for consumers of the package.  
The optimisation of the package internals can be dealt with at a later date.  Right now, the absence of the functionality this PR is adding is blocking all our other tickets.  
I'd be happy to open an issue that captures all of your concerns, Chris, and restructure the package internals later.

Reply:
Agreed with Sareh, however could we have the issue raised and commented here before the PR is merged. It could be labbeled with `alpha-3`

Reply:
Making one now